Ryujinx.HLE/HOS/Services/Fs/IFileSystemProxy.cs

```diff
@@ -125,6 +132,21 @@ public ResultCode OpenSdCardFileSystem(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [Command(30)]
+        // OpenGameCardStorage(u32, u32) -> object<nn::fssrv::sf::IStorage>
+        public ResultCode OpenGameCardStorage(ServiceCtx context)
+        {
+            GameCardHandle handle = new GameCardHandle(context.RequestData.ReadInt32());
+            GameCardPartitionRaw partitionId = (GameCardPartitionRaw) context.RequestData.ReadInt32();
+
+            Result rc = _baseFileSystemProxy.OpenGameCardStorage(out LibHac.Fs.IStorage storage, handle, partitionId);
```

Comment:
```suggestion
            Result resultCode = _baseFileSystemProxy.OpenGameCardStorage(out LibHac.Fs.IStorage storage, handle, partitionId);
```
and add an empty line after.